### PR TITLE
feat: slider widgets cypress test cases

### LINKED ISSUE #17466: [Task]: Cypress test cases for the slider widgets.
### Is there an existing issue for this?

- [X] I have searched the existing issues

### SubTasks

- Write test cases to cover all property pane validations.
- Write test case for checking the bindings of all slider widgets.

### PR DESCRIPTION
**Pull request by yaldram**
## Description

- Add cypress test cases for the slider widgets.

Fixes #17466 

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

- Ran all the cypress test cases locally.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: app/client/cypress/support/Pages/AggregateHelper.ts [hunk 1]
```diff
@@ -755,6 +756,12 @@ export class AggregateHelper {
       });
   }
 
+  public VerifyErrorMessage(errorMessage: string) {
```
Comment (Aishwarya-U-R) [inline app/client/cypress/support/Pages/AggregateHelper.ts:759]: Can this method name be '`VerifyEvaluatedErrorMessage`' - now it looks too generic.

**Review by Aishwarya-U-R — COMMENTED** [review]
@yaldram - Sucha neat work with respect to Cypress:)

File: app/client/cypress/support/Pages/AggregateHelper.ts (hunk 1 above)
Comment (yaldram) [reply app/client/cypress/support/Pages/AggregateHelper.ts]: Thanks a lot, makes sense. I have updated the method name to `VerifyEvaluatedErrorMessage`

**Review by Aishwarya-U-R — COMMENTED** [review]
Cypress looks good!

File: app/client/src/widgets/RangeSliderWidget/component/RangeSlider.tsx [hunk 2]
```diff
@@ -366,6 +366,7 @@ const RangeSliderComponent = (props: RangeSliderComponentProps) => {
             {...sharedThumbProps}
             disabled={disabled}
             dragging={active}
+            id="slider-thumb-one"
```
Comment (aswathkk) [inline app/client/src/widgets/RangeSliderWidget/component/RangeSlider.tsx:369]: @yaldram I believe this `id` is used only to target this element from Cypress test.
It's better to make use of something like `data-cy` for this use case.
ref: [Cypress best practises - Selecting Elements](https://docs.cypress.io/guides/references/best-practices#Selecting-Elements)

**Review by aswathkk — COMMENTED** [review]
@yaldram A minor comment regarding the best practise I found on Cypress docs. Otherwise everything looks good to me.

File: app/client/src/widgets/RangeSliderWidget/component/RangeSlider.tsx (hunk 2 above)
Comment (yaldram) [reply app/client/src/widgets/RangeSliderWidget/component/RangeSlider.tsx]: Thanks Aswath, appreciate it. I have updated the code to now use `data-cy` as the selector.

**Review by aswathkk — APPROVED** [review]

**Review by aswathkk — DISMISSED** [review]

**Review by aswathkk — APPROVED** [review]

**Comment by Sripriya93** [comment]
Working fine, moving to done.

**Review by aswathkk — APPROVED** [review]

**Pull request merged**